setup.py

```diff
@@ -27,6 +27,9 @@ def load_deps(file_name):
     "s3": ["s3fs~=0.5"],
     "az": ["adlfs~=0.5"],
     "kafka": ["kafka-python~=2.0"],
+    "redis": ["redis~=3.5"],
+    "redis-cluster": ["redis-py-cluster~=2.1.3"],
+    "lupa": ["lupa~=1.13"],
```

Comment:
Any reason that redis and lupa are pinned to a major version and redis-cluster is pinned to a minor version?

Reply:
no reason, fixed